### PR TITLE
Return parsed responseBody on error

### PR DESCRIPTION
**Pull request by jasonmadigan**

### CODE REVIEW

File: lib/base.js [hunk 1]
```diff
@@ -39,7 +39,14 @@ module.exports = function(params, cb){
...
     }
     if (response.statusCode.toString()[0]!=='2'){
-      return cb({body : body, code : response.statusCode, error : 'Non-2xx status code', url : fullUrl, method : params.method, body : params.body });
+      return cb({
+        responseBody: body,
+        code: response.statusCode,
+        error: 'Non-2xx status code',
+        url: fullUrl,
+        method: params.method,
+        body: params.body
```
Comment (jasonmadigan) [inline lib/base.js]: Previously body param was being clobbered by the request params body

**Comment by jasonmadigan** [comment]
@cianclarke you might review this one - I assume this is released via npm@fh?

**Comment by cianclarke** [comment]
Reviewed, looks great. Indeed published via npm feedhenry user

**Comment by jasonmadigan** [comment]
Published.

**Pull request merged**